extra-xfce/xfce4-settings/autobuild/defines

```diff
@@ -1,7 +1,7 @@
 PKGNAME=xfce4-settings
 PKGDES="Settings manager for Xfce"
 PKGDEP="exo libxfce4ui garcon libnotify libxklavier gtk-engines-2 \
-        gnome-themes-standard libcanberra sound-theme-freedesktop xf86-input-libinput"
+        gnome-themes-standard libcanberra sound-theme-freedesktop xf86-input-libinput xiccd"
 BUILDDEP="intltool"
 PKGSEC=xfce
 
```

Comment:
Is there any Autotools option required to enable `xiccd` support? If so, please *explicitly specify them* below in `AUTOTOOLS_AFTER=`.

Reply:
`xfce4-settings` provided a color management setting tool, it needs `xiccd` to work.

Reply:
Got it, this PR is good to go. Please package, upload, and tick off respective architectures.